VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
Yep. I think that should work. Testing it now.

On Thu, May 5, 2016 at 12:17 PM, Christopher Lee <notifications@github.com>
wrote:

> In VideoLocker/res/layout/fragment_course_combined_info.xml
> <https://github.com/edx/edx-app-android/pull/690#discussion_r62211460>:
>
> > @@ -21,7 +21,7 @@
> >          <RelativeLayout
> >              android:id="@+id/announcement_layout"
> >              android:layout_width="match_parent"
> > -            android:layout_height="wrap_content">
> > +            android:layout_height="fill_parent">
>
> Isn't the change here from wrap_content?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-app-android/pull/690/files/bd33bf34e28604b1e26eff7665bd542a84ec6bea#r62211460>
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX